todo_app/trello.py

```diff
@@ -0,0 +1,81 @@
+from types import MethodDescriptorType
+from dotenv import load_dotenv
+import requests
```

Comment:
You need to add this to the project's dependencies. Run `poetry add requests` to both install it in the virtual env and automatically update pyproject.toml appropriately, so that when other people run `poetry install` it will include it.

Otherwise others cannot run this project without knowing they need to manually install "requests" (and potentially what version?)